src/examples/radiuss_tutorial/lesson_00/README.md

```diff
@@ -0,0 +1,138 @@
+# Lesson 00: Details about our Axom configuration
+
+In this lesson, we will develop a simple application that uses an installed version of Axom. 
+Specifically, we will be print the version of Axom and some of its configuration properties.
+
+!!! tip  Our examples will be using the ``C++14`` standard since that's Axom's current version. 
+
+We will also be using the [fmt](https://fmt.dev/latest/index.html) library for string formatting, which Axom uses internally and vendors for its users.
+The vast majority of ``fmt`` has been included in recent C++ standards, starting with ``C++20``. See also the [fmt cheat sheet](https://hackingcpp.com/cpp/libs/fmt.html).
+
+## Configuring our application
+
+We will be using CMake to configure our examples against a pre-installed copy of Axom configured with several third-party libraries (TPLs) including the following RADIUSS libraries:
+* [Conduit](https://github.com/LLNL/conduit/)
+* [MFEM](https://mfem.org/)
+* [RAJA](https://github.com/LLNL/raja)
+* [Umpire](https://github.com/LLNL/umpire)
+
+Axom's build system is based on [BLT](https://github.com/LLNL/blt) -- a streamlined CMake-based foundation for <b>B</b>uilding, <b>L</b>inking and <b>T</b>esting large-scale high performance computing (HPC) applications.
+
+Since Axom users typically develop HPC applications that need to on several different platforms ("hosts") with several different compilers, we will be using a CMake cache file containing several configuration variables, including the compiler and MPI information and compilation flags. We refer to this file as a "host-config".
+
+Our application can be configured and built using the following commands:
+```shell
+> cd <path/to/tutorial/files>
+> mkdir build
+> cd build
+> cmake -C ../host-config.cmake ..
+> make -j16
+```
+
+!!! tip BLT provides a few lightweight "smoke" tests to help diagnose configuration issues. If we configure our application with the ``ENABLE_TESTS`` CMake option, the above command will build these tests and we can run them using ``ctest``.
```

Comment:
These ``!!! tip`` annotations look good in my VS-code environment. I will update them to github markdown after the tutorial. The latter has the form

\> \*\*note\*\*
\> A note here.

e.g.


> **note**
> A note here.

Reply:
I switched the alerts to things that can be rendered in github